### ISSUE TITLE
Cursor on Windows 10 hidden after boot

### ISSUE THREAD

**Issue by timo95** [talonvoice/talon#421]
Cursor is hidden after booting into Windows until I move the physical mouse. This is how it happens for me:
1. Activate "Start on login"
2. Reboot
3. Do not touch mouse while logging in (I use Windows Hello)
4. Activate control mouse -> Cursor is invisible (you can see the effect when hovering over buttons)
5. Move physical mouse -> Cursor is visible (and stays visible with control mouse)

This started appearing in one of the last few beta releases. I am not sure which exact version though.
Talon 0.2.0-154
Windows 10

**Comment by lunixbochs** [comment]
Can you see if this happens in the public version of Talon?

**Comment by timo95** [comment]
- Yes it does happen in the public version. Maybe it always happened and I didn't notice it before.
- It is even hidden without starting Talon or control mouse at all, but that wouldn't be a problem with a mouse.
- Windows Terminal shows a similar behavior: After entering any string the cursor is hidden when it is on top of the window until I move the physical mouse or interact with the window in any way (e.g. clicking on it).

**Comment by lunixbochs** [comment]
Do you have "Hide pointer while typing" enabled in your mouse settings?

**Comment by timo95** [comment]
Yes

**Comment by lunixbochs** [comment]
So am I right to say this is a feature request for Talon to attempt to force Windows to show the mouse cursor?

Another option may be for us to detect that a mouse cursor wouldn't be displayed, and show a little dot instead.

**Comment by timo95** [comment]
Yes, since that would make it easier to fully replace a physical mouse. Maybe the second option can be used for other modes than "Control Mouse", but here I would prefer to fully mimic a physical mouse (i.e. unhide the cursor on movement).

**Comment by lunixbochs** [comment]
See "The cursor isn't visible when I try moving" in this article.

https://support.parsec.app/hc/en-us/articles/115002623892-Mouse-and-Keyboard-Isn-t-Working-Correctly-When-Connected

```reg
Windows Registry Editor Version 5.00

[HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Policies\System]
"EnableCursorSuppression"=dword:00000000
```

I'm not sure if or when it makes sense for Talon to set this registry setting for you.
